drivers/gpu/drm/rp1/rp1-dsi/rp1_dsi_dsi.c

```diff
@@ -103,6 +103,24 @@
 
 /* And some bitfield definitions */
 
+#define DSI_PCKHDL_EOTP_TX_EN  BIT(0)
```

Comment:
Toss up as to whether it is better to put the bitfields alongside the register defines but indented, eg https://elixir.bootlin.com/linux/v6.10/source/drivers/gpu/drm/vc4/vc4_regs.h#L131

Reply:
I think I'll leave them where they are for now, otherwise I'd have to move the other (DPHY_* etc) definitions as well.